libraries/AP_Camera/AP_Camera_Logging.cpp

```diff
@@ -41,6 +42,16 @@ void AP_Camera_Backend::Write_CameraInfo(enum LogMessages msg, uint64_t timestam
         altitude_gps = 0;
     }
 
+    float_t mount_roll;
+    float_t mount_pitch;
+    float_t mount_yaw;
+    auto *s = AP_Mount::get_singleton();
```

Comment:
let's rename this "s" to "mount" and also add a space between the "if" and the bracket one line lower. txs!